### PR TITLE
Tooltip positions

### PR DESCRIPTION
**Pull request by surma**
Follow-up on #1101

### CODE REVIEW

**Comment by surma** [comment]
CLA has been signed by @kybarg in #1101

File: src/tooltip/README.md [hunk 1]
```diff
@@ -64,3 +64,7 @@ The MDL CSS classes apply various predefined visual enhancements to the tooltip.
 |-----------|--------|---------|
 | `mdl-tooltip` | Defines a container as an MDL tooltip | Required on tooltip container element |
 | `mdl-tooltip--large` | Applies large-font effect | Optional; goes on tooltip container element |
+| `mdl-tooltip--left` | Positiones the tooltip to the left of the target | Optional; goes on tooltip container element |
```
Comment (sgomes) [inline src/tooltip/README.md]: s/Positiones/Positions

Applies to other lines as well.

**Comment by sgomes** [comment]
Minor comment, LGTM otherwise!

File: src/tooltip/README.md (hunk 1 above)
Comment (surma) [inline src/tooltip/README.md]: :sweat_smile: Proof reading. What a concept...

**Comment by surma** [comment]
@sgomes Fix’d

**Comment by sgomes** [comment]
Thanks, merge at will :)

**Pull request merged**